app/repositories/UserRespository.ts

```diff
@@ -0,0 +1,22 @@
+import User from "../models/user";
+import { AppResponse } from "../services/app-responses";
+
+export class UserRespository {
+  public response: AppResponse = new AppResponse();
+
+  public findUserById = async (id) => {
+    return User.findById(id).exec();
```

Comment:
@Suresh-ashwini Any specific reason for using exec()..It works without that

Reply:
exec() removed from all db calls

Reply:
removed exec()